components/PageCard.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import { PageCardProps } from '../types';
+
+const PageCard = ({ cardInfo }: PageCardProps) => {
+  console.log(cardInfo);
+
+  return (
+    <>
+      {cardInfo.map((info, index) => {
+        return (
+          <div
+            key={index}
```

Comment:
Prefer using other data instead of array indexes for `key`, since it can lead to bugs. In this case, another unique piece of data for each element in the array could be the exercise title

Reply:
I have refactored it by adding the title